tests/atlas.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Atlas Connectivity Tests
+--SKIPIF--
+<?php
+if (!file_exists('.travis.scripts/atlas-uris.txt')) { echo "skip Atlas URIs not found\n"; }
+if ($_ENV['TESTS'] !== 'tests/atlas.phpt') { echo "skip Atlas tests not wanted\n"; }
```

Comment:
Do we still need a CI matrix entry that overrides `TESTS` to just run this Atlas test?

Reply:
To elaborate, normally the test suite is run with `make test` and `TESTS` is not explicitly set. Does this test only get run when running `make test TESTS=tests/atlas.phpt`?

Is there ever a case where `$_ENV['TESTS']` would not be set and this could raise a warning?

Reply:
Technically, this could warn. But skip-if code never throws a warning, as ``display_errors`` is disabled for SKIPIFs: https://github.com/php/php-src/blob/master/run-tests.php#L1626